cypress/e2e/applicant/apply.cy.js

```diff
@@ -0,0 +1,226 @@
+require("dotenv").config();
```

Comment:
do we need this line, since we also do this in `cypress.config.ts` - wouldn't that run first anyway?

Reply:
right you are - this was from before i started using Cypress.env and forgot to pull it out, thanks!